src/helpers/TextFormatting.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+export const formatTextOrLink = (text: string, urlTruncate?: number): JSX.Element => {
```

Comment:
Does make sense to have a component link instead of a helper? I think it would be more idiomatic than a helper. wdyt

Reply:
Good point